services/brig/brig.cabal

```diff
@@ -195,6 +195,8 @@ library
     , servant
     , servant-server
     , servant-swagger
+    , servant-swagger-ui
+    , servant-swagger-ui-core
```

Comment:
Just so we don't forget this adds dependency on the lzma c-library. This will need to be added to the dockerfile(s) and AMIs.

Reply:
Done.

Reply:
No, actually, this one: https://github.com/zinfra/hegemony/pull/1781 is also needed.